src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -850,6 +878,33 @@ export class ServerView extends DirMixin(PolymerElement) {
     this.dispatchEvent(makePublicEvent('RemoveAccessKeyRequested', {accessKeyId: accessKey.id}));
   }
 
+  setServerTransferredData(totalBytes) {
+    this.totalInboundBytes = totalBytes;
+  }
+
+  updateAccessKeyRow(accessKeyId, fields) {
+    let newAccessKeyRow;
+    if (accessKeyId === MY_CONNECTION_USER_ID) {
+      newAccessKeyRow = Object.assign({}, this.get('myConnection'), fields);
+      this.set('myConnection', newAccessKeyRow);
+    }
+    for (let ui in this.accessKeyRows) {
+      if (this.accessKeyRows[ui].id === accessKeyId) {
+        newAccessKeyRow = Object.assign({}, this.get(['accessKeyRows', ui]), fields);
+        this.set(['accessKeyRows', ui], newAccessKeyRow);
+        return;
+      }
+    }
+  }
+
+  _formatDataLimitForKey(key) {
+    return this._formatDisplayDataLimit(this._activeDataLimitForKey(key))
+  }
+
+  _formatDisplayDataLimit(limit) {
```

Comment:
This is hiding the dependency on localize. Pass as an input instead.

Reply:
Done in 67492a22a967c621796b3878b61831699324f2a4